filesystem/azure.go

```diff
@@ -0,0 +1,196 @@
+package filesystem
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"github.com/MSOpenTech/azure-sdk-for-go/storage"
```

Comment:
5kg suggest me use gofmt. 
The code is generated by gofmt.
I am confused about what the standard is.